docs/src/explanations/intrinsics.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: docs
+title:  "Intrinsics"
```

Comment:
There are 2 "table of contents" to add this -- the main explanations.md, and the menu.yml for the website. Can be done later if you don't mean to advertise this yet